gdb/riscv-tdep.c

```diff
@@ -1286,10 +1264,7 @@ riscv_gdbarch_init (struct gdbarch_info info,
   set_gdbarch_sw_breakpoint_from_kind (gdbarch, riscv_sw_breakpoint_from_kind);
 
   /* Register architecture.  */
-  set_gdbarch_pseudo_register_read (gdbarch, riscv_pseudo_register_read);
-  set_gdbarch_pseudo_register_write (gdbarch, riscv_pseudo_register_write);
   set_gdbarch_num_regs (gdbarch, RISCV_NUM_REGS);
-  set_gdbarch_num_pseudo_regs (gdbarch, RISCV_NUM_REGS);
```

Comment:
I did not check the default value for this field, but a defensive approach would be to explicitly initialise it at 0.

```c
// No pseudoregisters.
set_gdbarch_num_pseudo_regs (gdbarch, 0);
```